src/runtime/manifest.ts

```diff
@@ -102,6 +102,14 @@ export interface ManifestStack {
  * @internal
  */
 export function stackToWire(stack: ManifestStack): Record<string, unknown> {
+  if (stack.params) {
```

Comment:
In retrospect, I think there's a good argument to be made that this ugly special case should probably live in the generic toSpec() implementation in params/types.ts?

Reply:
nice "hack" with super great usability upgrade. Do you want to do that here or on a separate PR.